crates/light-client-prover/src/circuit/mod.rs

```diff
@@ -737,3 +755,16 @@ impl<S: Storage, DS: DaSpec, Z: Zkvm> Default for LightClientProofCircuit<S, DS,
         Self::new()
     }
 }
+
+/// Maps the Citrea network to a unique identifier used in the batch proof method ID upgrade transactions.
+pub fn citrea_network_to_method_id_upgrade_identifier(
+    network: sov_rollup_interface::Network,
+) -> u8 {
+    match network {
```

Comment:
Commenting on this again,

This method is NOT needed. If we use `repr(u8)` on the `Network` enum and then use `network as u8` we should be able to get the u8 value.

We can also hardcode the values we have here on the enum itself.